db/Articles.js

```diff
@@ -0,0 +1,81 @@
+//jshint esversion: 6
+
+///NEED TO EXAMINE THIS DB CLOSELY. What needs to be returned to the router in the instantiation?
+class Articles {
+  constructor() {
+    this.title = "";
+    this._collection = [];
+  }
+
+  getAllArticles(){
+    return this._collection;
+  }
+
+  getArticleByTitle(title) {
+    let index = null;
+    this._collection.forEach((object) => {
+      if(object.title === title) {
+        index = this._collection.indexOf(object);
+      }
+    });
+    if(index === null){
+      return false;
+    }else{
+      return this._collection[index];
+    }
+  }
+
+//article request has keys: title, body, author, urlTitle
+  addArticle(article){
```

Comment:
So it looks like you're either returning false or a collection of articles. It's probably best to only return one type either a boolean or an array but not the possibility of either or.